src/lib/flexbox/api/base.ts

```diff
@@ -127,7 +135,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     let value = this._lookupStyle(element, 'display');
 
     // Note: 'inline' is the default of all elements, unless UA stylesheet overrides
-    return value ? value.trim() : 'inline';
+    return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline');
```

Comment:
It will catch most cases, but not the ones that are set to `block` by the UA, which are:

p
h1, h2, h3, h4, h5, h6
ol, ul
pre
address
blockquote
dl
div
fieldset
form
hr
noscript
table